src/client/wrap.cpp

```diff
@@ -375,31 +375,231 @@ std::vector<GeneratedModel> ModelGenerator::generateAnotherModel(const std::vect
 using namespace pybind11::literals;
 
 PYBIND11_MODULE(pyprt, m) {
+	const char* docInit = "Initialization of PRT. PyPRT functionalities are blocked until the initialization is done.";
+	const char* docIsInit = R"mydelimiter(
+        is_prt_initialized() -> bool
+
+        This function returns *True* if PRT is initialized, *False* otherwise.
+
+        :Returns:
+            bool
+    )mydelimiter";
+	const char* docShutdown = "Shutdown of PRT. The PRT initialization process can be done only once per "
+	                          "session/script. Thus, ``initialize_prt()`` cannot be called after ``shutdown_prt()``.";
+	const char* docIs = R"mydelimiter(
+        __init__(*args, **kwargs)
+
+        The initial shape corresponds to the geometry on which the CGA rule will be applied.
+        )mydelimiter";
+	const char* docIsInitV = R"mydelimiter(
+        1. **__init__** (*vert_coordinates*)
+
+        Constructs an InitialShape with one polygon by accepting a list of direct vertex coordinates. The 
+        vertex order is expected to be counter-clockwise.
+
+        :Parameters:
+            **vert_coordinates** -- List[float]
+        :Example: ``shape1 = pyprt.InitialShape([0, 0, 0, 0, 0, 10, 10, 0, 10, 10, 0, 0])``
+        )mydelimiter";
+	const char* docIsInitVI = R"mydelimiter(
+        2. **__init__** (*vert_coordinates*, *face_indices*, *face_count*)
+
+        Constructs an InitialShape by accepting a list of direct vertex coordinates, a list of the vertex
+        indices for each faces and a list of the indices count per face. The vertex order is expected to 
+        be counter-clockwise.
+
+        :Parameters:
+            - **vert_coordinates** -- List[float]
+            - **face_indices** -- List[int]
+            - **face_count** -- List[int]
+        :Example: ``shape2 = pyprt.InitialShape([0, 0, 0, 0, 0, 10, 10, 0, 10, 10, 0, 0], [0, 1, 2, 3], [4])``
+        )mydelimiter";
+	const char* docIsInitP = R"mydelimiter(
+        3. **__init__** (*init_shape_path*)
+
+        Constructs an InitialShape by accepting the path to a shape file. This can be an OBJ file, Collada, etc.

```

Comment:
consider mentioning the supported file formats, i.e. the list of geometry decoders available on https://esri.github.io/esri-cityengine-sdk/html/esri_prt_codecs.html